_security/access-control/users-roles.md

```diff
@@ -138,6 +125,66 @@ See [YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#roles
 
 See [Create role mapping]({{site.url}}{{site.baseurl}}/security/access-control/api/#create-role-mapping).
 
+## Defining read-only roles
+
+A read-only role grants users the ability to read data from the OpenSearch cluster, but not to modify or delete any data. The read-only role is useful for scenarios where you want to provide access to data for reporting, analysis, or visualization purposes, without allowing modifications to the data or the cluster itself. This helps maintain data integrity and prevents accidental or unauthorized changes.
+
+As with any role in OpenSearch, read-only roles can be configured using the following three methods: 
+- Using OpenSearch Dashboards.
+- Modifying the `yml` configuration files.
+- Using the Cluster Settings API. 
+
+The most user friendly approach for anyone getting familiar with roles and role mappings is to use OpenSearch Dashboards, as it is easy to navigate the creation of roles and assign those roles to users. 
+{ .tip}
+
+### Defining a basic read-only role
+
+To create a basic read-only role which allows access to OpenSearch Dashboards, view existing dashboards, visualizations and query different indexes, use the following permissions.
+
+These permissions will give the user access to all tenants and indexes on the cluster.
+{: .note}
+
+#### Cluster permission
+
+For the user needs read-only access to cluster-wide resources, such as visualization or dashboards, add the `cluster_composite_ops_ro` permission to that user's role.
+
+#### Index permission
+
+If the user needs access to view visualizations, they will also need access to the index used to create the visualization. To give the user read-only access to all indexes, specify all (`*`) under the **Index** drop-down, and **Read** in **Index Permissions**.
+
+#### Tenant permissions
+
+If you use tenants to split work between different teams or projects, use the all (`*`) option followed by the **Read only** option.
+
+![creating role]({{site.url}}{{site.baseurl}}/images/role_creation_read_only.png)
+
+After all permission types are set and the role is created, you can directly map the role to a user by going to **Mapped users** tab in roles. Select **Map users** and select the user to map to this role. 
```

Comment:
```suggestion
After all permission types are set and the role is defined, you can map the role directly to a user in the **Mapped users** tab in roles. Select **Map users** and select the user to map to this role. 
```

Reply:
Is _roles_ a section in the UI? Then it should be capitalized. But is "in roles" really needed?